components/calendly_v2/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pipedream/calendly_v2",
+  "version": "1.0.0",
+  "description": "Pipedream Calendly_v2 Components",
+  "main": "dist/app/calendly_v2.app.mjs",
+  "keywords": [
+    "pipedream",
+    "calendly"
+  ],
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://pipedream.com/apps/calendly",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@pipedream/platform": "^1.5.1"
```

Comment:
If the imported package on get-event action `URL` is a npm package, you should add it here along with the expected version